### DISCUSSION TITLE
Changeable size for thumbnails

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#1437] [Ideas]
Is it possible to add a option via UI/config.yaml to change the size for created thumbnails in the Events list?

**Comment by mr-manuel** [comment]
For me it would also be nice to be able to set the thumbnail to a fixed width and/or percentual width on mobile to see more details scrolling through the list.

**Comment by Minglarn** [comment]
Would like to implement a settings in config for this as it's pretty hard to se the snapshots in mobile ui, so im bumping this.

**Comment by gfro84** [comment]
I support this. I've tried changing thumbnail to snapshot in the blueprint to no avail.

**Reply by NickM-27** [reply]
Not sure why you couldn't do this, I've done this and it works fine.

**Reply by gfro84** [reply]
Thanks, I don't want to distract from @mr-manuel original discussion but I'll give this another try on account of this. Will report back to @mr-manuel if it fills the UI better.

**Comment by NickM-27** [comment]
Are you talking about the size of the image used or just changing the size of the thumbnail in the UI to make the UI scale better?

**Comment by mr-manuel** [comment]
The thumbnail size itself (175 px * 175 px) for me can stay the same. I want only to change the display size. I know, that if the display size is bigger than the image size, the image will be pixeled, but this is no problem for me.
